lib/site-utils.js

```diff
@@ -86,6 +88,13 @@ exports.processPrice = function processPrice(priceStringInput) {
     _.includes(priceString, GBP_SYMBOL_PREFIX)) {
     logger.log('found gbp in price, converting to number...');
     price = accounting.unformat(priceString);
+  } else if (_.includes(priceString, INR_SYMBOL_PREFIX) ||
+    _.includes(priceString, INR_SYMBOL_PREFIX)) {
+    logger.log('found inr in price, converting to number...');
+    let tempPriceString = priceString;
+    tempPriceString = tempPriceString.substring(2, tempPriceString.length);
+    tempPriceString = tempPriceString.replace(',', '');
+    price = accounting.unformat(tempPriceString);
```

Comment:
I'm not familiar with rupees and how `accounting` handles them, but if we pass the `priceString` (not the `tempPriceString` that has been modified) to `accounting.uniformat`, will it handle it correctly?  Meaning could these lines instead be:
```
logger.log('found inr in price, converting to number...');
price = accounting.unformat(priceString);
```
Please take a look at [the accounting library](https://openexchangerates.github.io/accounting.js/), but the little I've tried works:
```
accounting.uniformat("R 9,444");  // returns 9444
```
What do you think?